config/webpack.dev.js

```diff
@@ -25,6 +25,7 @@ const HMR = helpers.hasProcessFlag('hot');
 // if env is 'inmemory', the inmemory debug resource is used
 const FABRIC8_FORGE_API_URL = process.env.FABRIC8_FORGE_API_URL || 'https://forge.api.prod-preview.openshift.io';
 const FABRIC8_WIT_API_URL = process.env.FABRIC8_WIT_API_URL || 'https://api.prod-preview.openshift.io/api/';
+const REALM = process.env.REALM || 'fabric8';
```

Comment:
The ENV var should be set in the build config the way "FABRIC8_BRANDING=openshiftio" is on https://github.com/fabric8io/fabric8-ui/blob/master/cico_build_deploy.sh#L27